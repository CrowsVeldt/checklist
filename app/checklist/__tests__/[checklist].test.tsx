import { render } from "@testing-library/react-native";
import Checklist from "../[checklist]";

describe("<HomeScreen />", () => {
  test("Checklist renders correctly", () => {
    const tree = render(<Checklist />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
