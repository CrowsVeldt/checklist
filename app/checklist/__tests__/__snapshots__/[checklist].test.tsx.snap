// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen /> Checklist renders correctly 1`] = `
<RNCSafeAreaView
  edges={
    {
      "bottom": "additive",
      "left": "additive",
      "right": "additive",
      "top": "additive",
    }
  }
>
  <Text />
</RNCSafeAreaView>
`;
